umpire-config.cmake.in

```diff
@@ -5,6 +5,5 @@
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
 get_filename_component(UMPIRE_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-set(UMPIRE_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
```

Comment:
Why did `UMPIRE_INCLUDE_DIRS` get removed?

Reply:
Because it seems to have accidentally been removed. I'll put it back!

Reply:
@jdahm, @davidbeckingsale, now I am wondering how it was working without `UMPIRE_INCLUDE_DIRS` being set.  Are there tests missing that should fail when this cmake variable isn't set?